frontend/coprs_frontend/coprs/views/apiv3_ns/apiv3_builds.py

```diff
@@ -76,11 +91,24 @@ def render_build(build):
     return flask.jsonify(to_dict(build))
 
 
-@apiv3_ns.route("/build/<int:build_id>/", methods=GET)
-@apiv3_ns.route("/build/<int:build_id>", methods=GET)
-def get_build(build_id):
-    build = ComplexLogic.get_build_safe(build_id)
-    return render_build(build)
+@apiv3_builds_ns.route("/<int:build_id>")
+class GetBuild(Resource):
+
+    @apiv3_builds_ns.doc(params=get_build_params)
+    @apiv3_builds_ns.marshal_with(build_model)
+    def get(self, build_id):
+        """
+        Get a build
+        Get details for a single Copr build.
+        """
+        build = ComplexLogic.get_build_safe(build_id)
+        result = to_dict(build)
+
+        # Workaround - `marshal_with` needs the input `build_id` to be present
+        # in the returned dict. Don't worry, it won't get to the end user, it
+        # will be stripped away.
```

Comment:
Don't you have some restx or other reference link?

Reply:
Honestly, I couldn't find any. This traceback happens when you remove the workaround:

```
werkzeug.routing.exceptions.BuildError: Could not build url for endpoint 'apiv3_ns.build_get_build' with values ['chroots', 'ended_on', 'id', 'is_background', 'ownername', 'project_dirname', 'projectname', 'repo_url', 'source_package', 'started_on', 'state', 'submitted_on', 'submitter']. Did you forget to specify values ['build_id']?
```

It doesn't make any sense to me, but adding the `build_id` key fixed the traceback. Of course, the `build_id` doesn't appear in the result because it is not defined in the schema, so it gets filtered out.